src/core/shared_state/currentMapPosition.ts

```
@@ -37,42 +39,59 @@ function jumpTo(map: Map, position: CenterZoomPosition) {
-          const { zoom } = camera;
-          const { lat, lng } = camera.center;
-          position = { ...position, lat, lng, zoom: zoom ?? map.getZoom() };
-          jumpTo(map, position);
-        }
-      }
-      state = position;
-    });
-
-    onAction('updateCurrentMapPosition', (position) => {
+    onAction('setState', (position) => {
       state = position;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)